.pre-commit-config.yaml

```diff
@@ -1,31 +1,30 @@
 files: bioio
 repos:
+    - repo: https://github.com/PyCQA/isort
+      rev: 5.12.0
+      hooks:
+          - id: isort
 
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.12.0
-    hooks:
-      - id: isort
+    - repo: https://github.com/myint/autoflake
+      rev: v2.2.0
+      hooks:
+          - id: autoflake
+            args: ["--in-place", "--remove-all-unused-imports"]
 
-  - repo: https://github.com/myint/autoflake
-    rev: v2.2.0
-    hooks:
-      - id: autoflake
-        args: ["--in-place", "--remove-all-unused-imports"]
+    - repo: https://github.com/psf/black
+      rev: 23.3.0
+      hooks:
+          - id: black
 
-  - repo: https://github.com/psf/black
-    rev: 23.3.0
-    hooks:
-      - id: black
+    - repo: https://github.com/PyCQA/flake8
+      rev: 6.1.0
+      hooks:
+          - id: flake8
+            additional_dependencies:
+                - flake8-typing-imports>=1.9.0
+                - flake8-pyprojecttoml
```

Comment:
Just as a general note, this is all fine but [ruff](https://github.com/astral-sh/ruff) does all of these in one except for `black`. My pre-commit configs are now: black, ruff, mypy

Reply:
The point of this PR is not to change any build or automation infrastructure.   But if you think we should use ruff then let's make a new ticket for that.

Reply:
Created a new issue: https://github.com/bioio-devs/bioio/issues/50

Reply:
I have been hearing good things.. 🦀